js/jquery.mapael.js

```diff
@@ -219,6 +219,11 @@
                 self.onUpdateEvent(e, opt);
             });
 
+            // Attach showElementsInRange event
+            self.$container.on("showElementsInRange." + pluginName, function (e, ranges) {
```

Comment:
Right! It was from an earlier version where I only passed a ranged array. I forgot to update this part after extending the event.